script/otbr-firewall

```diff
@@ -44,6 +44,8 @@ OTBR_FORWARD_INGRESS_CHAIN="OTBR_FORWARD_INGRESS"
 . /lib/lsb/init-functions
 . /lib/init/vars.sh
 
+set -euxo pipefail
```

Comment:
source _initrc instead?

Reply:
The script `otbr-firewall` will be installed into `/etc` and will be run as a service. In such a case I think it shouldn't rely on the source code.